src/locale/locales/fr/messages.po

```diff
@@ -49,15 +49,15 @@ msgstr "<0/> membres"
 
 #: src/view/com/auth/onboarding/RecommendedFeeds.tsx:30
 msgid "<0>Choose your</0><1>Recommended</1><2>Feeds</2>"
-msgstr "<0>Choisissez vos</0><1>fils d’actualité</1><2>recommandés</2>"
+msgstr "<0>Choisissez vos</0><1>recommandés</1><2>fils</2>"
```

Comment:
This sounds weird. We never use _recommandés fils_, always _fils recommandés_ in French. The `recommandé(s)` adjective is never written _before_ the noun in French, always _after_.